docs/map.json

```diff
@@ -2511,6 +2511,19 @@
       "id": "ci",
       "description": "Learn how to enable Distributed Tasks Executions, remote caching and more.",
       "itemList": [
+        {
```

Comment:
Weird that I cannot see the sidebar link even though it's in the `menu.json`. We must be filtering it out in the sidebar. (maybe because it's not top level?) We could make it `Getting Started -> Intro` like before.

<img width="1509" height="800" alt="Screenshot 2025-08-21 at 8 23 35 AM" src="https://github.com/user-attachments/assets/3ee28d3d-e347-4f5c-8ddd-bacf0e5b70b9" />

Reply:
forgot there was a sidebar allowlist. updated 👍